### PR TITLE
AXOL1TL change to v3 of model 

### PR DESCRIPTION
**Pull request by quinnanm**
#### PR description:

Small update to make default AXOL1TL model v3. 
cms-dist PR incorporating v3 of model: https://github.com/cms-sw/cmsdist/pull/8886
Trigger menu will be updated shortly with v3 seeds below.  ([updated menu example](https://github.com/quinnanm/cmssw/blob/emulator_25jan24/L1Trigger/L1TGlobal/data/Luminosity/startup/L1Menu_Collisions2023_v1_3_0_ADTv3.xml))

#### PR validation:

basic tests/code checks performed, checked that properly outputs v3 seeds when running full emulator on test file.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

N/A

### CODE REVIEW

File: L1Trigger/L1TGlobal/src/AXOL1TLCondition.cc [hunk 1]
```diff
@@ -90,11 +90,12 @@ const bool l1t::AXOL1TLCondition::evaluateCondition(const int bxEval) const {
...
 
   //HLS4ML stuff
-  std::string AXOL1TLmodelversion = m_AXOL1TLmodelversion;
+  std::string AXOL1TLmodelversion = m_AXOL1TLmodelversion;  //config loading method
+  // std::string AXOL1TLmodelversion = "L1Trigger/L1TGlobal/test/GTADModel_v3"; //for .so file in test dir loading method
   hls4mlEmulator::ModelLoader loader(AXOL1TLmodelversion);
   std::shared_ptr<hls4mlEmulator::Model> model;
   model = loader.load_model();
-  cout << "loading model... " << AXOL1TLmodelversion << std::endl;
+  // cout << "loading model... " << AXOL1TLmodelversion << std::endl;
```
Comment (aloeliger) [inline L1Trigger/L1TGlobal/src/AXOL1TLCondition.cc:98]: Could you remove the commented code if it doesn't really serve to demonstrate something please?

File: L1Trigger/L1TGlobal/src/AXOL1TLCondition.cc [hunk 2]
```diff
@@ -129,9 +130,9 @@ const bool l1t::AXOL1TLCondition::evaluateCondition(const int bxEval) const {
   ap_fixed<18, 13> EtSumInput[EtSumVecSize];
 
   //declare result vectors +score
-  std::array<ap_fixed<10, 7>, 13> result;
+  std::array<ap_fixed<10, 7, AP_RND_CONV, AP_SAT>, 8> result;  //v3
   ap_ufixed<18, 14> loss;
-  std::pair<std::array<ap_fixed<10, 7>, 13>, ap_ufixed<18, 14>>
+  std::pair<std::array<ap_fixed<10, 7, AP_RND_CONV, AP_SAT>, 8>, ap_ufixed<18, 14>>
```
Comment (aloeliger) [inline L1Trigger/L1TGlobal/src/AXOL1TLCondition.cc:135]: Not going to be picky about this, but maybe having some `typedefs` for different model versions of input might also be useful.

**Comment by quinnanm** [comment]
Thanks for the suggestions. I added the typedefs and removed unnecessary comments

**Comment by epalencia** [comment]
Please test

**Comment by aloeliger** [comment]
+l1

**Pull request merged**